captum/attr/_core/layer/layer_conductance.py

```diff
@@ -49,7 +49,7 @@ def attribute(
         target=None,
         additional_forward_args=None,
         n_steps=50,
-        method="riemann_trapezoid",
+        method="gausslegendre",
```

Comment:
Changing to `gausslegendre` in order to be consistent with IG and documentation.